garage/envs/base.py

```diff
@@ -61,6 +60,7 @@ def _close_viewer_window(self):
         search in those as well.
         This method can be removed once OpenAI solves the issue.
         """
+        from mujoco_py.mjviewer import MjViewer
```

Comment:
Is this the only solution? The CI / Ryan will probably complain about import formatting.

Reply:
nvm looks like Ryan approves

Reply:
Well, unless we had a MuJoCo-Env base class, I don't know where to move the method _close_viewer_window, but I'm open to suggestions since I also don't like to have imports at the middle of the code.
[This](https://travis-ci.com/rlworkgroup/garage/builds/87391938) build didn't complain.